File: Shop/Shop.Api.Core/Models/Product.cs
using System.Collections.Generic;

namespace Shop.Api.Core.Models
{
    public class Product
    {
        public long Article { get; set; }
        public string About { get; set; }
        public string Category { get; set; }
        public string Title { get; set; }
        public string Label { get; set; }
        public string Season { get; set; }
        public bool Gender { get; set; }
        public List<string> Photo { get; set; }
        public List<Sizes> SizesAvailable { get; set; }
        public double? Price { get; set; }
    }
}

File: API/pkg/models/product.go
package models

type Product struct {
	Article        int      `json:"article"`
	About          string   `json:"about"`
	Category       string   `json:"category"`
	Title          string   `json:"title"`
	Label          string   `json:"label"`
	Season         string   `json:"season"`
	Gender         bool     `json:"gender"`
	Photos         []string `json:"photo"`
	Price          float32  `json:"price"`
	SizesAvailable []Sizes  `json:"sizesAvailable"`
}

type Sizes struct {
	Size         float32      `json:"size"`
	Availability Availability `json:"availability"`
}

type Availability struct {
	IsAvailable     bool `json:"isAvailable"`
	QuantityInStock int  `json:"quantityInStock"`
}
